test/README.md

```diff
@@ -163,6 +163,11 @@ A stream to push an array into a REPL
 
 Blocks for `time` amount of time.
 
+### canCreateSymLink
+API to indicate whether the current running process can create
+symlinks. On Windows, this returns false if the process running
+doesn't have privileges to create symlinks (specifically [SeCreateSymbolicLinkPrivilege](https://msdn.microsoft.com/en-us/library/windows/desktop/bb530716(v=vs.85).aspx))
```

Comment:
I don't think we'll get that link below 80 chars, but if you put a newline before `[SeCreateSymbolicLinkPrivilege]` it should help.

Maybe also worth noting that it returns true for non-windows?